Client/game_sa/CStreamingSA.cpp

```diff
@@ -133,4 +133,10 @@ void CStreamingSA::RequestSpecialModel(DWORD model, const char* szTexture, DWORD
         call    dwFunc
         add     esp, 0xC
     }
-}
\ No newline at end of file
+}
+
+CStreamingInfo* CStreamingSA::GetStreamingInfoFromModelId(ushort id)
+{
+    CStreamingInfo* pItemInfo = (CStreamingInfo*)(ARRAY_StreamModelInfo);
+    return pItemInfo + id;
```

Comment:
I'd suggest using `return pItemInfo[id]`, as its an array

Reply:
This can't be compiled. Idk how to replace this

Reply:
Does this work?

```
CStreamingInfo[] pItemInfo = (CStreamingInfo[])(ARRAY_StreamModelInfo);
return pItemInfo[id]
```

Reply:
Invalid syntax 🤷‍♀️

Reply:
It's just an array of pointers, guys
```
CStreamingInfo** pItemInfo = (CStreamingInfo**)(ARRAY_StreamModelInfo);
return pItemInfo[id]
```

Reply:
`
CStreamingInfo* GetStreamingInfoFromModelId(uint id)
{
    CStreamingInfo* pItemInfo = (CStreamingInfo*)(0x8E4CC0);
    return &pItemInfo[id];
}
`